docs/apm/using-the-apm-ui.asciidoc

```diff
@@ -82,8 +120,8 @@ This page also displays a single sampled transaction, which is based on your sel
 You can update the sampled transaction by selecting a new _bucket_ in the transactions duration distribution graph.
 The number of requests per bucket is displayed when hovering over the graph, and the selected bucket is highlighted to stand out.
 
-//[role="screenshot"]
```

Comment:
Thoughts on whether or not this screenshot should exist? I'm having trouble deciding.

Reply:
I think it's valuable - that entire view consists of a lot of parts and many of them interactive, so it's not to break it up like this.